### PR TITLE
Support very old node

### PR DESCRIPTION
**Pull request by bcomnes**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # acorn5-object-spread changelog
 
+## 5.0.1
+* Remove arrow functions so this plugin works in super old node.
+* Add travis test way back to eternity.
```
Comment (ljharb) [inline CHANGELOG.md]: lulz

**Review by ljharb — APPROVED** [review]
LGTM, thanks!

**Comment by bcomnes** [comment]
@ljharb Any idea why npm I failed on 0.8?

**Comment by bcomnes** [comment]
Old npm.  Updated.

**Comment by ljharb** [comment]
Yup, LGTM

**Pull request merged**